app/components/article-comments.js

```diff
@@ -85,7 +85,7 @@ export default Component.extend(
 		init() {
 			this._super(...arguments);
 
-			this.set('model', ArticleCommentsModel.create({
+			this.set('model', ArticleCommentsModel.create(getOwner(this).ownerInjection(), {
```

Comment:
Maybe we could create our custom static method for creating models? Something like `ArticleCommentsModel .createModel(this, {data});`